force-app/main/default/quickActions/Funding_Request__c.Disbursment_Information.quickAction-meta.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
     <description>Action for display on lightning tab.</description>
-    <label>Disbursment Information</label>
+    <label>Disbursement Information</label>
```

Comment:
This is a production change.   Spell-fix.

Reply:
Ahh, okay. If it's not being used anymore, should we say that in the description or the name?

Reply:
@nicolersmith 
Oops, that's a typo - it's not that it not being used *anymore*, it's just not being used *anywhere* in our package. But a customer could totally be using it. It's basically just a component included in the package for the customer use if they want.